src/js/service/pensize/PenSizeService.js

```diff
@@ -2,7 +2,7 @@
   var ns = $.namespace('pskl.service.pensize');
 
   var MIN_PENSIZE = 1;
-  var MAX_PENSIZE = 4;
+  var MAX_PENSIZE = 1000000;
```

Comment:
(one additional comment about this: when the pen size gets really big, the rectangle/circle/line tools lag a lot - and the circle no longer resembles a circle. We should rework the tools so that they play nicely with bigger pen sizes.)